apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -460,6 +472,14 @@ type Storage struct {
 	// Whether this disk represents rotational storage
```

Comment:
Even if we cannot remove it, let's mark it as deprecated in the documentation.

Reply:
I don't think that it is deprecated, rather not recommended for usage.
Something like, use `that` field instead.

Reply:
Deprecated == not recommended for usage :) I'm fine with the proposed wording.

Reply:
So we need to update comments inside baremetal-operator or rather docs ?

Reply:
This comment since it's rendered in a few places.

Reply:
@dtantsur added comment about field deprecation.